runtime/onert/core/include/ir/TypeInfo.h

```diff
@@ -32,22 +33,40 @@ class TypeInfo
   TypeInfo() = delete;
 
   explicit TypeInfo(DataType type, float scale = 0, int32_t offset = 0)
-      : _type(type), _scale(scale), _offset(offset)
+      : _type(type), _scale(scale), _offset(offset), _sparse(false), _w0_size(0)
   {
   }
 
 public:
   DataType type() const { return _type; }
   float scale() const { return _scale; }
   int32_t offset() const { return _offset; }
+  bool sparse() const { return _sparse; }
+  int w0_size() const { return _w0_size; }
+  const uint16_t *w1_segments() const { return _w1_segments.data(); }
```

Comment:
In tflite spec, this type can be INT32, UINT16, and UINT8. In my opinion, TypeInfo should support all three sparse types.

Reply:
@periannath Yes, we can do later if necessary. Note that we don't support all operator and input data type in tflite specification.